src/fu-quirks.h

```diff
@@ -144,6 +144,20 @@ const gchar	*fu_quirks_lookup_by_usb_device		(FuQuirks	*self,
  */
 #define	FU_QUIRKS_DFU_AVR_CHIP_ID		"fwupd-dfu-avr-chip-id"
 
+/**
+ * FU_QUIRKS_FLASHROM_HWID:
+ * @key: a HWID GUID, e.g. `a0ce5085-2dea-5086-ae72-45810a186ad0`
+ * @value: a device ID, e.g. `librem15v3`
+ *
+ * When the system motherboard can be updated using flashrom, this quirk maps
+ * a hardware ID to a device ID.
+ *
```

Comment:
Flashrom can be used for more than just system motherboards can't it?  It might make sense to be more general in this regard.  For example add a quirk that represents system motherboard, and then a different one that represents some other random device (not sure what arguments `flashrom `uses to represent access to something other than `internal:laptop `though).

Reply:
I've never seen flashrom update anything other than the BIOS in the wild -- I know it can do things like network cards, but I'm really pushing vendors to use UEFI for that kind of thing -- flashrom is a last ditch effort really.

Reply:
OK probably fine to keep it this way and if the need arises it can be changed some then.